ethcore/sync/src/lib.rs

```diff
@@ -53,8 +53,10 @@ extern crate enum_primitive;
 extern crate macros;
 #[macro_use]
 extern crate log;
+extern crate parity_util_mem;
+extern crate parity_util_mem as mem;
```

Comment:
these two lines should be removed, right?

Reply:
alas, they are required by the derive macro (the second one, the first one is used in the imports), just double checked.